### PR TITLE
StripeAPI setup

### PR DESCRIPTION
**Pull request by jtingling**
### What this PR does (required):
- Initial setup StripeAPI.
- Setup development account, publishable key environment variable on front-end.

### Any information needed to test this feature (required):
- Install dependencies.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
+.sample.env
```
Comment (moffatethan) [inline .gitignore:17]: We wouldn't want to ignore this file ever - if other developers were to get onboarded it's easier to show what environment variables they need to run the app. 

It's just important not to expose any credentials in that file

File: client/src/.env.development.local.sample [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_STRIPE_PUBLISHABLE_KEY=pk_test_51KJhK0ANCC3EPL26kStEMdsyYC7mI0DmPelXzz3OkZK1tMnpPgy07ZFFFwdbbNxKaJMvfnoxu1m7wGar5qIxFMe600GH1LLEQr
```
Comment (moffatethan) [inline client/src/.env.development.local.sample:1]: So, even though this is a public key - I still would advise against putting it in the sample client env far. But good thinking on adding that file here!

**Review by moffatethan — CHANGES_REQUESTED** [review]
Overall good, let's revert some changes and just make sure to not expose the public and secret key on the repo.

File: .gitignore (hunk 1 above)
Comment (jtingling) [reply .gitignore]: I have reverted and removed the keys but kept the file.

File: client/src/.env.development.local.sample (hunk 2 above)
Comment (jtingling) [reply client/src/.env.development.local.sample]: Same as above.

**Review by moffatethan — APPROVED** [review]

**Pull request merged**